src/CMakeLists.txt

```diff
@@ -87,10 +87,10 @@ set(mudlet_SRCS
     dlgVarsMainArea.cpp
     EAction.cpp
     exitstreewidget.cpp
-    FontManager.cpp
     FileOpenHandler.cpp
+    FontManager.cpp
     GifTracker.cpp
-    TrailingWhitespaceMarker.cpp
+    GMCPAuthenticator.cpp
```

Comment:
Are we doing away with the convention of putting a 'T' in front of our classes? Does it have a meaning?

Reply:
For the meaning I am not sure but the `T` prefix could stand for a "template" (but not a C++ template) for a class that *isn't* associated with a dialogue (those begin with `dlg` instead!) Incidentally when describing a container or method for an arbitrary class `T` on it's own is often used as the identifier of the class. E.g. `(bool) QListIterator<T>::hasNext()` to test to see whether an iterator into a `QList<T>` has a value it can get. So maybe appending things to that seemed logical, `QListIterator<TMapLabel>::hasNext()` when iterating though a `QList<TMapLabel>`?

Reply:
I think so. I don't see a meaning behind it other than a `T`emplate which we don't use often.